lambda/main.go

```diff
@@ -86,7 +95,11 @@ func uploadFileToS3(session *session.Session, uploadFileDir string) error {
 		ContentDisposition:   aws.String("attachment"),
 		ServerSideEncryption: aws.String("AES256"),
 	})
-	return err
+	if err != nil {
```

Comment:
Not required. `return err` will take care of both i.e return nil or err if available

Reply:
updated @namandf